.github/workflows/update_internal_libraries.yml

```diff
@@ -59,3 +60,55 @@ jobs:
             
             This PR updates the reference tests dependency to the latest available version and copies the necessary files.
             - [ ] All tests must pass
+
+  autofill:
+    name: Update Autofill
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Install npm-check-updates
+        run: npm install -g npm-check-updates
+
+      - name: Install copy-files-from-to
+        run: npm install -g copy-files-from-to
+
+      - name: Check for updates
+        id: update-check
+        continue-on-error: true
+        run: ncu @duckduckgo/autofill --errorLevel 2
+
+      - name: Update reference tests
```

Comment:
Not sure what these two steps each specifically do ("Check for updates" and "Update reference tests") but I assume

- both are correct
- the latter needs renamed

@marcosholgado can you rename/clarify?

Reply:
Check for updates as its name explains checks if there are any new versions for autofill. The latter needs renaming and I've just done it :)

Reply:
> Check for updates as its name explains checks if there are any new versions for autofill so if not the workflow finished there. The latter needs renaming and I've just done it :)